tests/e2e/improved_csi_idempotency.go

```diff
@@ -344,6 +510,48 @@ func createVolumeWithServiceDown(serviceName string, namespace string, client cl
 
 		ginkgo.By(fmt.Sprintf("Sleeping for %v seconds to allow full sync finish", fullSyncWaitTime))
 		time.Sleep(time.Duration(fullSyncWaitTime) * time.Second)
+	} else if serviceName == hostdServiceName {
+		ginkgo.By("Fetch IPs for the all the hosts in the cluster")
+		hostIPs := e2eVSphere.getAllHostsIP(ctx)
+		isServiceStopped = true
+		for index, hostIP := range hostIPs {
+			if index%2 == 0 {
+				ginkgo.By(fmt.Sprintf("Stopping hostd service on the host  %s ...", hostIP))
+				sshCmd := fmt.Sprintf("/etc/init.d/hostd %s", stopOperation)
+				op, err := runCommandOnESX("root", hostIP, sshCmd)
+				framework.Logf(op)
```

Comment:
move this to `runCommandOnESX()` method itself

Reply:
Fixed